fix(l10n): translations update from weblate.recipesage.com

recipesage-weblate opened the pull request:
Translations update from [RecipeSage Weblate](https://weblate.recipesage.com) for [RecipeSage App/Frontend UI](https://weblate.recipesage.com/projects/recipesage-app/app-frontend/).



Current translation status:

![Weblate translation status](https://weblate.recipesage.com/widget/recipesage-app/app-frontend/horizontal-auto.svg)

Francois389 committed:
fix(l10n): translated using Weblate (French)

Currently translated at 98.8% (1077 of 1090 strings)

Translation: RecipeSage App/Frontend UI
Translate-URL: https://weblate.recipesage.com/projects/recipesage-app/app-frontend/fr/

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/julianpoy/RecipeSage?pullRequest=1944) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/julianpoy/RecipeSage?pullRequest=1944) before we can accept your contribution.<br/>**1** out of **5** committers have signed the CLA.<br/><br/>:white_check_mark: YoSafeguard<br/>:x: Francois389<br/>:x: mxc2135<br/>:x: Leukoplasthea<br/>:x: matiasso<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/julianpoy/RecipeSage?pullRequest=1944) it.</sub>

Leukoplasthea committed:
fix(l10n): translated using Weblate (German)

Currently translated at 100.0% (1096 of 1096 strings)

Translation: RecipeSage App/Frontend UI
Translate-URL: https://weblate.recipesage.com/projects/recipesage-app/app-frontend/de/

YoSafeguard committed:
fix(l10n): translated using Weblate (Russian)

Currently translated at 100.0% (1096 of 1096 strings)

Translation: RecipeSage App/Frontend UI
Translate-URL: https://weblate.recipesage.com/projects/recipesage-app/app-frontend/ru/

mxc2135 committed:
fix(l10n): translated using Weblate (Spanish)

Currently translated at 100.0% (1096 of 1096 strings)

Translation: RecipeSage App/Frontend UI
Translate-URL: https://weblate.recipesage.com/projects/recipesage-app/app-frontend/es/

matiasso committed:
fix(l10n): translated using Weblate (Finnish)

Currently translated at 89.0% (976 of 1096 strings)

Translation: RecipeSage App/Frontend UI
Translate-URL: https://weblate.recipesage.com/projects/recipesage-app/app-frontend/fi/

matiasso committed:
fix(l10n): translated using Weblate (Finnish)

Currently translated at 99.7% (1093 of 1096 strings)

Translation: RecipeSage App/Frontend UI
Translate-URL: https://weblate.recipesage.com/projects/recipesage-app/app-frontend/fi/

matiasso committed:
fix(l10n): translated using Weblate (Finnish)

Currently translated at 100.0% (1096 of 1096 strings)

Translation: RecipeSage App/Frontend UI
Translate-URL: https://weblate.recipesage.com/projects/recipesage-app/app-frontend/fi/
